Data/Infrastructure/OJS.Data.Infrastructure/DateTimeValueConverter.cs

```diff
@@ -0,0 +1,41 @@
+namespace OJS.Data.Infrastructure;
+
+using Common.Utils;
+using Microsoft.EntityFrameworkCore.Storage.ValueConversion;
+using System;
+using System.Linq.Expressions;
+using static Common.GlobalConstants.EnvironmentVariables;
+
+public class DateTimeValueConverter : ValueConverter<DateTime?, DateTime?>
+{
+   private static readonly TimeZoneInfo TimeZoneInfo = TimeZoneInfo.FindSystemTimeZoneById(EnvironmentUtils.GetRequiredByKey(LocalTimeZone));
```

Comment:
Rename to `LocalTimeZoneInfo` to differentiate this from the System's TimeZoneInfo class